### PR TITLE
super simplified the module to only using promise, and only offering a *...

### PR DESCRIPTION
**Pull request by sole**
...connect\* functionality

### CODE REVIEW

**Comment by sole** [comment]
hey @tofumatt @brittanystoroz r? (no emoji now, I'm tired)

File: examples/startAndConnect.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var connect = require('../');
+var startSimulator = require('node-firefox-start-simulator');
+
+startSimulator({}).then(function(simulator) {
```
Comment (tofumatt) [inline examples/startAndConnect.js:6]: Is the empty options hash required? I don't think it needs to be in the example here :-)

**Comment by tofumatt** [comment]
I'm gonna push some proposed tweaks to this but no reason not to merge now. :+1:

File: examples/startAndConnect.js (hunk 1 above)
Comment (sole) [inline examples/startAndConnect.js:6]: there is a bug in startSimulator https://github.com/mozilla/node-firefox-start-simulator/issues/8

**Pull request merged**